app/lib/application_extension.rb

```diff
@@ -19,6 +19,14 @@ module ApplicationExtension
     before_destroy :close_streaming_sessions, prepend: true
   end
 
+  class_methods do
+    def webapp
+      Doorkeeper::Application.find_by!(name: 'Web', superapp: true)
```

Comment:
Previous finders did just rely on `superapp: true`, but we likely want to match on name as well, if not using a fixed ID for the webapp.

Reply:
Are there other places where we query on the name or superapp value here? (that were not fully pulled in by this PR) ... if so, maybe do this via some reusable scopes in this module?

Unrelated, you can (I think?) probably omit the base class here, since this module is only ever included into that model? Maybe I'm missing something there.

Reply:
I checked, but these seemed to be the only cases